css-layout-classes/styles.css

```diff
@@ -0,0 +1,27 @@
+.container {
+  max-width: 800px;
+  margin: 0, auto;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.row {
+  display: flex;
+  flex-direction: row;
+  flex-basis: 100%;
```

Comment:
`flex-basis` is a child property, so it doesn't get applied to flex parents. This has no impact, so it should be removed